draft-ietf-rtcweb-jsep.xml

```diff
@@ -2581,6 +2586,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>The "a=rtcp-mux" line MUST only be added if
                         present in the most recent answer.</t>
 
+			<t>An "a=rtcp-mux-only" line, as specified in
```

Comment:
Hmm, the more I think about this, I think we should say iff the policy is require, and the other side does not have mux-only in it's SDP (both answer and offer), then we generate an error of some type that rejects the SDP and incompatible. I'd much rather have failure with an explicit reason than just the circuit breakers fail down the road or something